src/niamoto/cli/commands/stats.py

```
@@ -0,0 +1,605 @@
+                            FROM occurrences
+                            WHERE {col} IS NOT NULL
+                        """)
+                        row = result.first()
+                        if row and row.count_non_null > 0:
+                            stats["Numerical Data"][col] = {
+                                "Count": f"{row.count_non_null:,}",
+                                "Range": f"{row.min_val:.2f} - {row.max_val:.2f}",
+                                "Average": f"{row.avg_val:.2f}",
+                            }
+            except Exception:
+                pass
```

Pylint (3.3.7)

[refactor] Too many nested blocks (6/5)

(R1702)